chaos_kitten/brain/postman_parser.py

```
@@ -0,0 +1,310 @@
+        }
+        
+    def get_endpoints(self) -> List[Dict[str, Any]]:
+        """Get the list of extracted endpoints.
+        
+        Returns:
+            List[Dict[str, Any]]: List of normalized endpoint objects.
+        """
+        if not self._endpoints:
+             try:
+                self.parse()
+             except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)